### DISCUSSION TITLE
Cadet, and child type characters bug out after environment like step 4

### DISCUSSION THREAD

**Discussion by Pulparch** [bcholmes/StarTrek2d20#198] [General]
Hi I have tried on three computers, and my phone, I get like a black screen after choosing species, origin, and environment when choosing either cadet or child character types.

Also is there a way to randomly roll a solar system or sector?

**Comment by bcholmes** [comment]
There's a random sector generator. It only appears if you have included the "Shackleton Expanse" source, because many of the rules originate with that source.
